Code/Tools/ProjectManager/Source/ProjectSettingsScreen.cpp

```diff
@@ -145,8 +145,13 @@ namespace O3DE::ProjectManager
         ValidateProjectName() && ValidateProjectPath();
     }
 
-    bool ProjectSettingsScreen::Validate()
+    AZ::Outcome<void, QString> ProjectSettingsScreen::Validate()
     {
-        return ValidateProjectName() && ValidateProjectPath();
+        if (ValidateProjectName() && ValidateProjectPath())
+        {
+            return AZ::Success();
+        }
+
+        return AZ::Failure<QString>("");
```

Comment:
might be good to add a comment here that by returning an empty string error message we are opting to use the default